docs/Contributing/reference/configuration-for-contributors.md

```diff
@@ -154,6 +154,19 @@ This is the content of the PEM-encoded private key for the Apple Business Manage
       -----END RSA PRIVATE KEY-----
   ```
 
+### mdm.sso_rate_limit_per_minute
+
+The number of requests per minute allowed to [Initiate SSO during DEP enrollment](https://github.com/fleetdm/fleet/blob/main/docs/Contributing/reference/api-for-contributors.md#initiate-sso-during-dep-enrollment) and
+[Complete SSO during DEP enrollment](https://github.com/fleetdm/fleet/blob/main/docs/Contributing/reference/api-for-contributors.md#complete-sso-during-dep-enrollment) endpoints, combined.
```

Comment:
```suggestion
[Complete SSO during DEP enrollment](https://github.com/fleetdm/fleet/blob/main/docs/Contributing/reference/api-for-contributors.md#complete-sso-during-dep-enrollment) endpoints, combined.

The best practice is to set this to the number of new employees (end users) that onboard at the same time (ex. `100` if 100 end users setup their Macs simultaneously).
```

I think let's throw a best practice in here. That way, anyone who comes across this new config will understand why it's in here.

Reply:
@noahtalerman Should this be moved to the public configuration documentation instead?

Reply:
Lets do x3 as a just in case